app/src/main/java/eu/kanade/tachiyomi/data/backup/models/BackupCovers.kt

```diff
@@ -0,0 +1,16 @@
+package eu.kanade.tachiyomi.data.backup.models
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.protobuf.ProtoNumber
+
+@Serializable
+data class BackupCovers(
+    @ProtoNumber(1) val mappings: List<BackupCover> = emptyList(),
+)
+
+@Serializable
+data class BackupCover(
+    @ProtoNumber(1) val mangaId: Long,
```

Comment:
Imo it's better to just store the associated filename instead of the id
```suggestion
    @ProtoNumber(1) val filename: String,
```